src/redux/books/books.js

```diff
@@ -0,0 +1,37 @@
+// Define constants
+const ADDBOOk = 'ADD_BOOk';
+const REMOVEBOOK = 'REMOVE_BOOK';
```

Comment:
- [ ] Great job on all the functionality but here I think it would be great if you Define action types for adding and removing a book. in DUCK pattern so I recommend this to be somthing like this you can check this [article](https://github.com/erikras/ducks-modular-redux) for more info ❤✅🔥✨

> const ADD_BOOK = bookstore/books/ADD_BOOK
> const REMOVE_BOOK = bookstore/books/REMOVE_BOOK